kf_lib_data_ingest/common/constants.py

```diff
@@ -14,6 +14,8 @@ class COMMON:
     NOT_AVAILABLE = "Not Available"
     NOT_REPORTED = "Not Reported"
     NO_MATCH = "No Match"
+    # New: Needed to support Data Tracker templates
+    NOT_PROVIDED = "Not Able to Provide"
```

Comment:
I think I would favor
```suggestion
    NOT_ABLE_TO_PROVIDE = "Not Able to Provide"
```
since not provided and not able to provide don't mean the same thing to me